automation.py

```diff
@@ -345,26 +352,23 @@ def count_graph(directory, filename):
 
 
 def generate_summaries(directory):
-    try:
-        os.stat('{}/summaries'.format(directory))
-    except FileNotFoundError:
+    mode = os.lstat('{}/summaries'.format(directory)).st_mode
+    if not stat.S_ISDIR(mode):
         os.mkdir('{}/summaries'.format(directory))
-    else:
-        print(Exception())
-        logger.error('Error with summaries folder.')
-        exit(1)
 
     for root, _, files in os.walk(directory):
         for filename in files:
-            regex = re.compile(r'([\w-]+)_([\w-]+).(\w.+)')
+            regex = re.compile(r'([\w-]+)_([\w-]+)_([0-9]+).(\w.+)')
             matches = regex.match(filename)
 
             if 'summaries' not in root and regex.match(filename) and \
-                    matches.group(3) == 'json':
-                _summary_name = "{}/summaries/{}_{}-summary.json".format(
+                    matches.group(4) == 'json':
+                print(filename)
```

Comment:
is this print supposed to be in here? it seems like something from testing but i might just be missing context of it.

Reply:
Thanks!